ChecklistsUpdated/ChecklistViewController.swift

```diff
@@ -7,45 +7,33 @@
 //
 
 import UIKit
-//test
+
 class ChecklistViewController: UITableViewController {
 
     private struct Constant {
         static let NumberOfRows = 100
         static let CellReuseID = "ChecklistItem"
         static let CellLabelTag = 1000
-
     }
     
     override func tableView(_ tableView: UITableView,
-                               numberOfRowsInSection/*external parameter name*/ section/*local parameter name*/: Int) -> Int {
+                               numberOfRowsInSection section: Int) -> Int {
+
         return Constant.NumberOfRows
     }
     
     override func tableView(_ tableView: UITableView,
-                            
-                            // Below cellForRowAt method is to deliver a new or recycled cell object to the table view when a row becomes visible.
-                            cellForRowAt indexPath: IndexPath) -> //IndexPath is simply an object that points to a specific row in the table. When the table view asks the data source for a cell, you can look at the row number inside the indexPath.row property to find out the row for which the cell is intended.
+                            cellForRowAt indexPath: IndexPath) ->
```

Comment:
Minor formatting thing but I usually but the function return type on the same line as the `->`. So, `cellForRowAt indexPath: IndexPath) -> UITableViewCell {`